web/styles/compose.css

```diff
@@ -987,11 +1015,10 @@ textarea.new_message_textarea,
         color: inherit;
         text-decoration: none;
         font-size: 22px;
-        border-radius: 3px;
 
         &:hover {
             opacity: 1;
-            background-color: hsl(0deg 0% 0% / 10%);
+            background-color: hsl(0deg 0% 0% / 5%);
```

Comment:
Should this color be made a variable? It shows up elsewhere in the PR, too.

Reply:
Though this exact color does not, a near identical one is used for the same element in the dark mode. I've extracted it as a variable and defined it for both light and dark modes in `app_variables.css`